radboudumc-template/templates/new/publication-thesis.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+{% block title %}{{ page.title }}{%endblock%}
+{% block description %}{{ page.content | striptags | truncate(200)}}{% endblock %}
+{% block page_picture %}{{SITEURL}}/images/theses/{{ page.coverpng }}{% endblock %}
+{% block content %}
+    {% include 'blocks/breadcrumbs.html' %}
+
+    <div class="container">
+    <h1 class="page-title">{{ page.title }}</h1>
+
+    <p class="lead">
+        {% for author in page.authors %}
+            {{ author }}
+        {% endfor %}
+    </p>
+    <img src="{{SITEURL}}/images/theses/{{ page.coverpng }}">
```

Comment:
Please use the image util snippet here to make sure that the images are in the right size.